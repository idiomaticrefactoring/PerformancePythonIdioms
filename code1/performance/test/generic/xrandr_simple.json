{
 "screens": [
  {
   "screen_number": 0,
   "minimum_width": 8,
   "minimum_height": 8,
   "current_width": 1920,
   "current_height": 1080,
   "maximum_width": 32767,
   "maximum_height": 32767,
   "associated_device": {
    "associated_modes": [
     {
      "resolution_width": 1920,
      "resolution_height": 1080,
      "is_high_resolution": false,
      "frequencies": [
       {
        "frequency": 60.03,
        "is_current": true,
        "is_preferred": true
       },
       {
        "frequency": 59.93,
        "is_current": false,
        "is_preferred": false
       }
      ]
     },
     {
      "resolution_width": 1680,
      "resolution_height": 1050,
      "is_high_resolution": false,
      "frequencies": [
       {
        "frequency": 59.88,
        "is_current": false,
        "is_preferred": false
       }
      ]
     }
    ],
    "is_connected": true,
    "is_primary": true,
    "device_name": "eDP1",
    "rotation": "normal",
    "resolution_width": 1920,
    "resolution_height": 1080,
    "offset_width": 0,
    "offset_height": 0,
    "dimension_width": 310,
    "dimension_height": 170
   }
  }
 ],
 "unassociated_devices": []
}