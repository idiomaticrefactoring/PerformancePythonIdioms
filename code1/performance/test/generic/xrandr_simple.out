Screen 0: minimum 8 x 8, current 1920 x 1080, maximum 32767 x 32767
eDP1 connected primary 1920x1080+0+0 (normal left inverted right x axis y axis) 310mm x 170mm
   1920x1080     60.03*+  59.93
   1680x1050     59.88
